docs/sources/project/review-pr.md

```diff
@@ -72,15 +79,15 @@ To update your existing pull request:
         #		modified:   docs/sources/installation/mac.md
         #		modified:   docs/sources/installation/rhel.md
 
-4. Push to your origin.
+5. Force push the change to your origin.
 
-        $ git push origin
+        $ git push -f origin <branch_name>
```

Comment:
We use $ to indicate an actual command line.  That isn't the case here. This is a syntax example.  Can you remove that please?